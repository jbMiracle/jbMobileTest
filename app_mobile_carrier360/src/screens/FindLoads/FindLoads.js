import React, { Component } from 'react';
import { View } from 'react-native';
import { Text, connectStyle } from 'native-base';
import styleDef from './FindLoads.style';

class FindLoads extends Component {
  render() {
    return (
      <View testID='findLoads' style={ { flex: 1, justifyContent: 'center', alignItems: 'center' } }>
        <Text>Find Loads</Text>
      </View>
    );
  }
}

export default connectStyle('C360.FindLoads', styleDef)(FindLoads);
