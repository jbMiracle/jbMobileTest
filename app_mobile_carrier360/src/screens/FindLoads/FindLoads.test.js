import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import FindLoads from './FindLoads';

it('can render an empty component', () => {
  const renderer = new ShallowRenderer();

  renderer.render(<FindLoads />);

  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
