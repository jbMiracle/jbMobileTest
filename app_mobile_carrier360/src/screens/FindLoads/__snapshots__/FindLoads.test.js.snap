// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render an empty component 1`] = `
<FindLoads
  style={Object {}}
  virtual={undefined}
/>
`;
